index.js

```diff
@@ -60,17 +75,6 @@ function Babel(inputTree, _options) {
   // delete it from the options hash
   delete this.options.helperWhiteList;
 
-  if (this.options.browserPolyfill) {
-    var babelCorePath = require.resolve('@babel/core');
-    babelCorePath = babelCorePath.replace(/\/babel-core\/.*$/, '/babel-core');
-
-    let polyfill = funnel(babelCorePath, { files: ['browser-polyfill.js'] });
-    this.inputTree = mergeTrees([polyfill, inputTree]);
-  } else {
-    this.inputTree = inputTree;
-  }
-  delete this.options.browserPolyfill;
-
```

Comment:
was there a particular reason for moving the block above the `Filter.call()`?

Reply:
Yeah, the code is building a modified input tree which must be passed into the super class' constructor. Modifying `this.inputTree` after calling the constructor doesn't work (anymore?).